### PR TITLE
updated graph views

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: app/models/label.py [hunk 1]
```diff
@@ -70,6 +70,10 @@ class Label(db.Model, ModelMixin):
     gift: orm.Mapped[str] = orm.mapped_column(sa.String(128), default="", nullable=True)
 
     user: orm.Mapped[User] = orm.relationship("User", backref="labels")
+    _views: orm.Mapped[list[LabelView]] = orm.relationship(
+        "LabelView",
```
Comment (nixissmtp) [inline app/models/label.py:74]: remove LabelView

File: app/models/label_view.py [hunk 2]
```diff
@@ -6,6 +8,9 @@
 from app.database import db
 from .utils import ModelMixin
 
+if TYPE_CHECKING:
+    from .label import Label
+
 
 def generate_uuid() -> str:
```
Comment (nixissmtp) [inline app/models/label_view.py:15]: remove

File: app/templates/base.html [hunk 3]
```diff
@@ -31,6 +31,9 @@
   <!-- prettier-ignore -->
   {% block links %}
   {% endblock %}
+  <script src="https://unpkg.com/htmx.org@1.9.9"
```
Comment (nixissmtp) [inline app/templates/base.html:34]: hash htmx lib

File: app/templates/report/label_views_modal.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="flex justify-center">
+            <button type="submit"
+                class="mt-3 px-5 py-2 text-base font-medium text-center text-white bg-indigo-700 rounded-lg hover:bg-indigo-800 focus:ring-4 focus:ring-blue-300 sm:w-auto dark:bg-indigo-600 dark:hover:bg-indigo-700 dark:focus:ring-blue-800">
+                Export to Excel
+            </button>
+        </div>
+
+    </div>
+
+    <script id="script-label-view-{{ unique_id }}">
```
Comment (nixissmtp) [inline app/templates/report/label_views_modal.html:36]: make script for close modal as macros

**Review by nixissmtp — CHANGES_REQUESTED** [review]

File: app/templates/macros/close_modal.html
Comment (sa1am8) [inline app/templates/macros/close_modal.html:1]: maybe hide this in .ts file?

**Review by sa1am8 — APPROVED** [review]

**Pull request merged**